design/sketch.webidl

```diff
@@ -489,7 +491,7 @@ interface WebGPUProgrammablePassEncoder {
     WebGPUCommandBuffer endPass();
     // Allowed in both compute and render passes
     //TODO: setPushConstants() ?
-    void setBindGroup(u32 index, WebGPUBindGroup bindGroup);
+    void setBindGroup(u32 index, WebGPUBindGroup bindGroup, sequence<u32>? dynamicOffsets);
```

Comment:
`dynamicOffsets` should be optional, not just nullable. (Ideally it would be non-nullable and default to the empty array, but I don't know if that's efficient in JS?)